tests/splitting.test.js

```diff
@@ -126,6 +126,29 @@ function test(recordExtractor) {
       it.eq(records[0].text, 'one two three'));
   });
 
+  it.describe('page with characters that need escaping', it => {
+    const words = Array.from({ length: 9000 }, (...k) => `"word${k}"`); // each word is surrounded by double quotes. each will be stored as two characters. (\")
```

Comment:
The generated words are `"word,0"`, `"word,1"`, ... is this what you wanted?

And why do you surround them by `"` actually?

Reply:
as I tried to convey in the title of this test, I added double quotes to make sure that the splitting respects the max record size limit, even if the text contains characters that would be escaped when stringified into JSON. see https://github.com/algolia/crawler-configurations-examples/pull/3#discussion_r356478178 for more explanations.

about the coma, it's probably caused by the fact that I'm using `(...k)` to get the second parameter of the function. the actual content of those items don't matter that much but you're right, I'm gonna get rid of those commas.

Reply:
I just removed the commas, to avoid any distraction.